packages/mui-codemod/src/deprecations/select-classes/test-cases/expected.css

```diff
@@ -0,0 +1,11 @@
+.MuiSelect-filled ~ .MuiSelect-icon {
```

Comment:
`icon` component is rendered as sibiling to `select` component, hence used `~` instead of ` > `